app.js

```diff
@@ -22,3 +22,4 @@ serverController.addEndpoint('/api*', new APIController(
 ));
 // The static page controller serves the basic form of the pages
 serverController.addEndpoint('/*', new StaticPageController());
+serverController.startServer(process.env.PORT);
```

Comment:
Do you need a fallback here in case `process.env.PORT` isn't set? That might happen if some random developer (not Heroku/App Engine) launches it with `node app.js` from the command line, right?